src/qtiCommonRenderer/renderers/interactions/TextEntryInteraction.js

```diff
@@ -132,7 +132,7 @@ var render = function render(interaction) {
 
             hideTooltip($input);
 
-            if (regex.test($input.val())) {
+            if ($input.val().length && regex.test($input.val())) {
```

Comment:
Won't do.
Take a look at the changes, you've introduced in the last PR https://github.com/oat-sa/tao-item-runner-qti-fe/pull/60/files
You'll be back to the same state where you started from.